### PR TITLE
Created Product listing page

### PR DESCRIPTION
**Pull request by anushasunil**
image

### CODE REVIEW

**Review by shivangisharma01 — COMMENTED** [review]
The code is clean and easily understandable. Good use of variable. As such no changes required.

**Review by shivangisharma01 — APPROVED** [review]
Looks good to merge.

File: Product-listing/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+    }
+
+    .input-box, .brand-name,.brand-info, .sitemap, .contact-you {
+        flex-grow: 1;
+    }
+
+    /* .product-display {
+        grid-template-columns: 1fr;
+    } */
+
```
Comment (dev-enforced) [inline Product-listing/product-listing.css]: Can remove these commented lines of code

**Review by dev-enforced — APPROVED** [review]

File: Product-listing/product-listing.css (hunk 1 above)
Comment (anushasunil) [reply Product-listing/product-listing.css]: Thanks for pointing this out

**Pull request merged**